_sass/_blocks.scss

```diff
@@ -2739,10 +2742,11 @@ blockquote:after {
   }
 
   p {
-    color: $pink;
+    color: #ffffff;
```

Comment:
Is there no `$white` variable or should we create one for white? 

Same applies to the blue (#1D214F) used above - if it's a commonly used colour used throughout the website?

Reply:
@a-bresler let me look into this.

Reply:
@a-bresler  done. Thank you for the feedback.

Reply:
![giphy](https://user-images.githubusercontent.com/25475628/132532227-caeaeb0e-7583-40f1-ad0c-27dfa8d8ae80.gif)